report_thesis/src/sections/background.tex

```diff
@@ -93,7 +93,24 @@ \subsubsection{Principal Component Analysis (PCA)}
 where $\mathbf{V}_n$ is the matrix that only contains the top $n$ eigenvectors.
 
 \subsubsection{Kernel PCA}
-% \citet{scholkopftKPCA}
+The description of the \gls{kernel-pca} algorithm is based on \citet{learningwithkernels}.
+\gls{kernel-pca} is an extension of the traditional \gls{pca} that is used to handle nonlinear relationships among data points.
+The core idea behind \gls{kernel-pca} is to map the data into a higher-dimensional space using a kernel function, known as the \textit{kernel trick}. 
+This mapping allows for the linear separation of data points in the higher-dimensional space, even when the data is not linearly separable in the original space.
+
+The core element of \gls{kernel-pca} is to obtain the covariance matrix in the higher-dimensional space. 
+Mathematically, this is given by:
+$$
+	\mathbf{C} = \frac{1}{m} \sum_{j=1}^{m} \Phi(\mathbf{x}_j) \Phi(\mathbf{x}_j)^\top
+$$
+where $m$ is the number of data points and $\Phi(\mathbf{x}_j)$ represents the mapping of the data point $\mathbf{x}_j$ into the higher-dimensional feature space. 
```

Comment:
I believe this entire part is wrong (at least based on my understanding and what you write afterwards). In traditional PCA, the covariance matrix captures the variance and covariance between every pair of features in the dataset, which is crucial for identifying the principal components that capture the most variance in the data. But, as you say, this is not feasible in practice for large datasets - hence the kernel trick.